sources/packages/web/src/components/aest/students/modals/AddManualOverawardDeduction.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <v-form ref="addManualOverawardForm">
+    <modal-dialog-base title="Add manual record" :showDialog="showDialog">
+      <template #content>
+        <error-summary :errors="addManualOverawardForm.errors" />
+        <p class="label-value">
+          Add a record to capture that the student paid back their loans through
+          the National Student Loans Service Centre (NSLSC).
+        </p>
+        <v-select
+          label="Award"
+          density="compact"
+          :items="awardTypeItems"
+          v-model="formModel.awardValueCode"
+          variant="outlined"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Award Type')]"
+        />
+        <v-text-field
+          hide-details="auto"
+          density="compact"
+          label="Add the dollar amount"
+          v-model="formModel.overawardValue"
+          variant="outlined"
+          type="number"
+          prefix="$"
+          :rules="[
+            (v) => checkNullOrEmptyRule(v, 'Overaward deduction amount'),
+            (v) =>
+              numberRangeRule(
+                v,
+                -MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE,
+                MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE,
+                'Overaward deduction amount',
```

Comment:
The `formatCurrency` can be added here to have the numbers formatted as "money".
If possible, remove the "(CAD)" from the `formatCurrency` method.